packages/assets/index.js

```diff
@@ -0,0 +1 @@
+exports.packageLocation = __dirname;
```

Comment:
I haven't found a better way yet to get the package location, as we need to copy files based on that

Reply:
Package location is not the pkg.json, right?

Reply:
It's the location of the folder